fetch_firmware.sh

```diff
@@ -28,11 +28,11 @@ if [[ $(uname -a) == *NixOS* ]]; then
 fi
 
 BR_VERSION=$(git describe --abbrev=0 --tags)
-FW_VERSION=${1:-v2.1.21}
+FW_VERSION=${1:-v2.1.21-1-g4239d51d}
```

Comment:
Is the plan to make a new v2.1 release at some point so that this doesn't have to point to a non-release artifact?

Reply:
Yes I will update next time the release script is run.

Reply:
Never mind, I see that this was merged prior to the v2.1.22 tag: https://github.com/swift-nav/piksi_firmware_private/commit/4239d51d9ec81edfe742281ce66df762c0e3ed2a